packages/fxa-shared/test/subscriptions/metadata.ts

```diff
@@ -48,6 +49,129 @@ const requiredProductMetadata = {
   webIconURL: null,
 };
 
+const PLAN_WITH_METADATA: AbbrevPlan = {
```

Comment:
Just noticed `AbbrevPlan` here. I've removed it and cleaned up all the new variables that are no longer necessary.